src/background-script/setup.ts

```diff
@@ -63,6 +63,8 @@ import { TabManager } from 'src/tab-management/background/tab-manager'
 import { ReadwiseBackground } from 'src/readwise-integration/background'
 import { type } from 'openpgp'
 import pick from 'lodash/pick'
+import ActivityStreamsBackground from 'src/activity-streams/background'
+import { string } from 'prop-types'
```

Comment:
I think this import is unintentional - vscode did it to me a few times

Reply:
Ah yes, that was is unintentional.